src/libs/actions/Session.js

```diff
@@ -143,9 +143,10 @@ function fetchAccountDetails(login) {
  * re-authenticating after an authToken expires.
  *
  * @param {String} authToken
+ * @param {String} encryptedAuthToken
```

Comment:
I think it's worth adding in the docs here that this isn't necessary for the API call to create a login, but it's used in setSuccessfulSignInData